src/osd/PrimaryLogPG.cc

```diff
@@ -4245,8 +4245,11 @@ static int check_offset_and_length(uint64_t offset, uint64_t length, uint64_t ma
 {
   if (offset >= max ||
       length > max ||
-      offset + length > max)
+      offset + length > max) {
+    std::cerr << __func__ << " "
```

Comment:
i don't think the `stderr` is available after the ceph-osd is daemonized. please use `ldpp_dout()` for printing the log messages in a static function.